packages/clay-button/src/ClayButton.soy

```diff
@@ -135,8 +129,12 @@
 
 	{if $ariaLabel}
 		aria-label="{$ariaLabel}"
+	{elseif $label}
```

Comment:
You need to add `label` and `icon` as parameters of `.attributes` template and pass'em from `.render`

Reply:
Yes, I got a failing build and I've amended this by now, thanks for the head-ups though.